INSTALL.rst

```diff
@@ -16,7 +16,17 @@ Matplotlib and its dependencies are available as wheel packages for macOS,
 Windows and Linux distributions::
 
   python -m pip install -U pip
-  python -m pip install -U matplotlib
+  python -m pip install --no-user --prefer-binary -U matplotlib
+
+The ``--prefer-binary`` flag will select the newest version of
+Matplotlib for which there is a wheel.  Remove this flag if you are
+willing to possibly compile Matplotlib from source.
+
+As of pip 20.1, if the :file:`site-packages` folder is not writable,
+it will fallback to behaving as if ``--user`` was passed.  In some
+cases this behavior is desirable, it can also lead to very confusing
+installation issues.  If you are comfortable with that risk, then
+remove this flag.
```

Comment:
"remove the ``--no-user`` flag." might be more clear instead of "remove this flag."

Reply:
Done!